plugins/flytekit-kf-pytorch/flytekitplugins/kfpytorch/task.py

```diff
@@ -471,7 +471,11 @@ def fn_partial():
         except ChildFailedError as e:
             _, first_failure = e.get_first_failure()
             if is_recoverable_worker_error(first_failure):
-                raise FlyteRecoverableException(e.format_msg())
+                ex = FlyteRecoverableException(e.format_msg())
+                # keep the timestamp of the original exception, rather than
```

Comment:
Following [this](https://github.com/flyteorg/flytekit/pull/2797/files#r1805800708) comment, I would have passed the timestamp to the constructor here.